frontend/packages/dev-console/src/components/import/import-submit-utils.ts

```diff
@@ -190,6 +197,79 @@ export const createBuildConfig = (
   return k8sCreate(BuildConfigModel, buildConfig, dryRun ? dryRunOpt : {});
 };
 
+export const createDeployment = (
```

Comment:
I wonder if we couldn't use the `shared-submit-utils` so we don't go down the same path of duplication as before.

Reply:
I had the same feeling. But I don't want to disturb the existing code flow at the last moment. I have to take care for few bugs. I'll refactor this code along with that.

Reply:
Please log a refactor task @gijohn

Reply:
https://jira.coreos.com/browse/ODC-2136